packages/stream-model-instance-handler/src/__tests__/model-instance-document-handler.test.ts

```diff
@@ -303,7 +303,7 @@ describe('ModelInstanceDocumentHandler', () => {
     expect(modelInstanceDocumentHandler.name).toEqual('MID')
   })
 
-  it('makes genesis commits correctly', async () => {
+  /*it('makes genesis commits correctly', async () => {
```

Comment:
Please, add a comment on top like `TODO NET-XXXX Description`, where `XXXX` is a reference to a corresponding Linear ticket, and `Description` is a short rationale for commenting this out.

Reply:
Fixed an issue on my end and tests work as expected (added back in).